www/docs/en/dev/cordova/storage/storage.md

```diff
@@ -166,6 +166,7 @@ IndexedDB is supported by the underlying WebView on the following Cordova platfo
 
 - Windows (with some limitations)
 - Android (4.4 and above)
+- iOS (with Safari 10 and above)
```

Comment:
```suggestion
- iOS 10 and above
```
I would change the iOS line to something like this, because

1. People might not know the safari version matches the iOS version
2. Sounds like it's only supported in Safari, but it's also supported on the WebView and apps that use WebView (Cordova apps, Chrome, etc)